### DISCUSSION TITLE
Change tmdb overview language

### DISCUSSION THREAD

**Discussion by Nonoss117** [sbondCo/Watcharr#729] [Ideas]
Hello,
English is not my first language, and I'd like to change the language of the overview (summary) of movies and tv shows

And may be more, ability to change ui language :-)
Thank's

**Comment by IRHM** [comment]
Hi @Nonoss117 thank you for commenting on this. It looks like we have an issue for this in #442 hopefully we will be getting around to it soon.

The overviews/titles will likely come first, then we will be looking at adding localization to our app text. Hoping its not a long wait, but there are some pressing things i'd like to get to first.

Thanks!
